### ISSUE TITLE
Narrator Does Not Read Field Labels When Tabbing Through Inputs in Import Code Wizzard UI

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/nodejstools#1367]
## Steps
1. Enable windows narrator.
1. Create a new NTVS project and select `JavaScript -> From Existing Code`
1. Use tab to navigate between text input boxes in the window.

![image](https://cloud.githubusercontent.com/assets/12821956/19537278/393f451a-9604-11e6-8ee5-71727486fa3d.png)

## Expected
When a text box is tabbed to, the narrator reads out what the text box is used for.

## Actual
Narrator reads:

> Editing CONTENTS_OF_TEXT_BOX

You can use caps lock + left arrow to select and read the label for the text input, and then tab back to the input, but I believe the narrator should read what the  input is used for when tabbing through the inputs.
